daemons/gptp/common/ptp_message.cpp

```diff
@@ -803,9 +805,13 @@ bool PTPMessageFollowUp::processMessage(MediaDependentPort *port, Timestamp rece
 		-(TIMESTAMP_TO_NS(corrected_sync_time - system_time));
 
 	offset.master_time = TIMESTAMP_TO_NS(preciseOriginTimestamp);
+	//printf("Master Time: %llu\n", offset.master_time);
+	//printf("System Time: %llu\n", TIMESTAMP_TO_NS(system_time));
```

Comment:
Do we need to keep this commented code? Do we already have a standardized of way of enabling/disabling debugs?